### PR TITLE
Feature/icinga

### PR DESCRIPTION
**Pull request by TasDeniz**
Change icinga project value by docker environment

### CODE REVIEW

File: src/docker/samply.share.broker.conf [hunk 1]
```diff
@@ -13,3 +13,4 @@ icinga.username={icinga-username}
 icinga.password={icinga-password}
 # (A) for dev system (B) for production
 icinga.site_suffix={icinga-site-suffix}
+icinga.project=-{icinga-project}
```
Comment (alexanderkiel) [inline src/docker/samply.share.broker.conf:16]: does the `=-` has special meaning here or is it just a typo?

File: src/main/java/de/samply/share/broker/rest/Searchbroker.java [hunk 2]
```diff
@@ -240,108 +240,6 @@ public Response getAllDirectoryIdOptions() {
     }
   }
 
-  /**
```
Comment (alexanderkiel) [inline src/main/java/de/samply/share/broker/rest/Searchbroker.java:243]: is this related to the feature?

File: src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java [hunk 3]
```diff
@@ -61,9 +62,11 @@
   private static String siteSuffix;
   private static CredentialsProvider credentialsProvider;
   private static HttpClientContext context;
+  private static String project;
```
Comment (alexanderkiel) [inline src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java:65]: would move this near siteSuffix

File: src/docker/start.sh [hunk 4]
```diff
@@ -30,6 +30,7 @@ sed -i "s#{icinga-path}#${ICINGA_PATH}#"                /usr/local/tomcat/conf/s
 sed -i "s/{icinga-username}/${ICINGA_USERNAME}/"        /usr/local/tomcat/conf/samply.share.broker.conf
 sed -i "s/{icinga-password}/${ICINGA_PASSWORD}/"        /usr/local/tomcat/conf/samply.share.broker.conf
 sed -i "s/{icinga-site-suffix}/${ICINGA_SITE_SUFFIX}/"  /usr/local/tomcat/conf/samply.share.broker.conf
+sed -i "s/{icinga-project}/${ICINGA_PROJECT}/"          /usr/local/tomcat/conf/samply.share.broker.conf
```
Comment (alexanderkiel) [inline src/docker/start.sh:33]: can you please add documentation of this new env var?

File: src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java [hunk 5]
```diff
@@ -193,11 +193,8 @@ private static void sendPerformanceReport(String sitename, StatusReportItem stat
       boolean isRetry) throws IcingaConnectorException {
     try {
       HttpPost httpPost;
-      if (ProjectInfo.INSTANCE.getProjectName().toLowerCase().equals("samply")) {
-        httpPost = createPost(sitename, statusReportItem.getParameterName() + "-gba");
-      } else {
-        httpPost = createPost(sitename, statusReportItem.getParameterName());
-      }
+      httpPost = createPost(sitename, statusReportItem.getParameterName()
+          + project);
```
Comment (alexanderkiel) [inline src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java:197]: shouldn't we pass the minus sign here? or should the project start with a minus? I assume the project should be just called `gba`

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
If there is no documentation regarding the icinga env vars, it would be important to have one.

File: src/main/java/de/samply/share/broker/rest/Searchbroker.java (hunk 2 above)
Comment (TasDeniz) [reply src/main/java/de/samply/share/broker/rest/Searchbroker.java:243]: No, just wanted to delte unnecessary code

File: src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java (hunk 5 above)
Comment (TasDeniz) [reply src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java:197]: ok

**Review by alexanderkiel — APPROVED** [review]
